core/src/main/res/values-eu/strings.xml

```diff
@@ -41,7 +41,7 @@
   <string name="pref_back_to_top">Gora itzuli</string>
   <string name="pref_back_to_top_summary">Bistaratu botoi bat orrialdearen amaieran goiko aldera mugitzeko</string>
   <string name="pref_hide_toolbar">Tresna-barra ezkutatu desplazatzean</string>
-  <string name="pref_hide_toolbar_summary">\u2022 On GNU/Linux and with the console: split —bytes=4000M my_big_file.zim</string>
+  <string name="pref_hide_toolbar_summary">u2022 On GNU/Linux and with the console: split —bytes=4000M my_big_file.zim</string>
```

Comment:
the English source for this is
```
Keeps the toolbar from scrolling out of view when browsing content
```
and removing the `\` here stops it becoming a Unicode character, this will just display `u2022`

Reply:
@Abijeet

Reply:
Hmm, this looks like this is an issue on our end. This was removed during an import - https://translatewiki.net/w/i.php?title=Kiwix%3AAndroid.ui.pref_hide_toolbar_summary%2Feu&type=revision&diff=9243340&oldid=9108321 after this should change was made - https://github.com/kiwix/kiwix-android/commit/581b127184259ae999bd3501d95bb908aaffa1e5#diff-c533bde4f259f63373679ba0e169ac32R44

This is obviously wrong and needs some investigation.

Reply:
In the meantime I am pretty happy to merge this if you want to approve the PR.

While this string is in error I don't think we have enough basque users to cause confusion

Reply:
Created [T244501](https://phabricator.wikimedia.org/T244501) to track this. I'll resolve this, and approve the PR.

Thanks for reporting these issues, and for your patience.

Reply:
I've fixed this translation. This should be exported during the next export on Monday, 10th Feb.